src/Akeneo/Bundle/ElasticsearchBundle/Cursor/SearchAfterBoundedCursor.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Akeneo\Bundle\ElasticsearchBundle\Cursor;
+
+use Akeneo\Bundle\ElasticsearchBundle\Client;
+use Akeneo\Component\StorageUtils\Cursor\CursorInterface;
+use Akeneo\Component\StorageUtils\Repository\CursorableRepositoryInterface;
+
+/**
+ * Bounded cursor to iterate over items where a start and a limit are defined.
```

Comment:
from an external point of view, they have the same behavior
that's why the first sentence of the description is the same

not the others, because internally, it's not the same the same mechanism that is used to achieve this goal